Tests/_TestData-PowerPlatform/PowerAppOnlySolution/CanvasApps/src/TestApp/DataSources/customers %28v2.0%29.json

```diff
@@ -0,0 +1,45 @@
+[
```

Comment:
File name is bit weird.

Reply:
I took the file structure and name from a sample project - to make sure it is as realistic as possible.

The file structure and naming for Power Apps is something that is handled in the Power Platform